infercnvpy/tl/_copykat.py

```diff
@@ -0,0 +1,106 @@
+import infercnvpy as cnv
+import numpy as np
+import pandas as pd
+import scipy
+from scipy.sparse import issparse
+from anndata import AnnData
+from scanpy import logging, AnnData
+
+def copyKAT(
+    adata: AnnData,
+    key_added: str = "copyKAT",
+    inplace: bool = True,
+    layer: str = None
+)-> pd.DataFrame:
+    """Inference of genomic copy number and subclonal structure.
+
+    Runs CopyKAT (Copynumber Karyotyping of Tumors) based on integrative 
+    Bayesian approaches to identify genome-wide aneuploidy at 5MB resolution 
+    in single cells to separate tumor cells from normal cells, and tumor 
+    subclones using high-throughput sc-RNAseq data.
+    
+    Note: The matrix values are often the count of unique molecular identifier (UMI) 
+    from nowadays high througput single cell RNAseq data. The early generation of 
+    scRNAseq data may be summarized as TPM values or total read counts, 
+    which should also work.
+
+    You can find more info on GitHub: https://github.com/navinlabcode/copykat
+
+    DOI: https://doi.org/10.1038/s41587-020-00795-2
```

Comment:
Could you please add this to `docs/references.bib`? 

Then you can do `` :cite:`WhateverBibtexId` `` somewhere in the text and it gets formatted as citation.

Reply:
done